mod_reforged/hooks/ui/screens/tooltip/tooltip_events.nut

```diff
@@ -0,0 +1,37 @@
+::Reforged.HooksMod.hook("scripts/ui/screens/tooltip/tooltip_events", function(q) {
+	q.general_queryUIElementTooltipData = @(__original) function( _entityId, _elementId, _elementOwner )
+	{
+		switch (_elementId)
+		{
+			case "character-screen.left-panel-header-module.Experience":
+				return [
+					{
+						id = 1,
+						type = "title",
+						text = "Experience"
+					},
+					{
+						id = 2,
+						type = "description",
+						text = ::Reforged.Mod.Tooltips.parseString("Characters gain experience as they or their allies slay enemies in battles. If a character has accumulated sufficient experience, he\'ll [level up|Concept.Level] and be able to increase his [attributes|Concept.CharacterAttribute] and pick a [perk|Concept.Perk] that grants a unique bonus.\n\nBeyond the 11th character level, characters are veterans and gain perk points only every 4th level. They can still improve their attributes but the attribute gain per level is small.")
```

Comment:
Now that we are rewriting this tooltip we should use the existing vanilla variable `::Const.XP.MaxLevelWithPerkpoints` instead of writing out the number 11.

This is one step closer to having a smooth transition for our mod or submods when they want to change this value.

Reply:
This is a good point. I will adjust the code accordingly.